Source/DotSpatial.Analysis/VectorToRaster.cs

```diff
@@ -128,7 +128,8 @@ public static IRaster ToRaster(IFeatureSet fs, Extent extent, double cellSize, s
                         PolygonScheme ps = new PolygonScheme();
                         colorTable = ps.GenerateUniqueColors(fs, fieldName);
                         mpl.Symbology = ps;
-                        mpl.DrawRegions(args, new List<Extent> { env });
+                        mpl.DrawRegions(args, new List<Extent> { env }, false);
+                        mpl.DrawRegions(args, new List<Extent> { env }, true);

```

Comment:
The first draws the normal colors of the features and the second draws the selection color of the selected features on top.